utils/bccmd/Makefile

```
@@ -0,0 +1,44 @@
+# SPDX-Identifier-License: GPL-2.0-or-later
+#
+# Copyright (C) 2023 Vincent Tremblay <vincent@vtremblay.dev>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bccmd
+PKG_VERSION:=6.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER=Vincent Tremblay <vincent@vtremblay.dev>
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_FIXUP:=autoreconf
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/bccmd/bccmd.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=2cd7e6eb755785101a60e1da70de2d7788ba7bd2459b576e82b5851ed422588e
```

```suggestion
PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/bccmd/bccmd.git
PKG_SOURCE_VERSION:=v$(PKG_VERSION)
PKG_MIRROR_HASH:=2cd7e6eb755785101a60e1da70de2d7788ba7bd2459b576e82b5851ed422588e

PKG_MAINTAINER=Vincent Tremblay <vincent@vtremblay.dev>
PKG_LICENSE:=GPL-2.0-or-later
PKG_LICENSE_FILES:=COPYING

PKG_FIXUP:=autoreconf
```

consider adding `PKG_BUILD_PARALLEL:=1` if this program can be compiled multithreaded.

---

done, I fixed the order as suggested and added the `PKG_BUILD_PARALLEL` config, thanks!